action.yml

```
@@ -0,0 +1,44 @@
+name: 'Build Plugin'
+description: 'Build GOG Galaxy integration on current OS'
+
+inputs:
+  src:
+    description: 'Location of directory containing manifest.json and essential plugin files'
+    required: true
+    default: 'src'
+  build:
+    description: 'Relative path to expected build directory' 
+    required: true
+    default: 'build'
+  requirements:
+    description: 'Location of Python requirements file'
+    required: true
+    default: 'requirements/app.txt'
+  third-party-target:
+    description: 'Target directory where pip packages are dumped; relative to `build`'
+    required: false
+    default: '.'
+  python-arch:
+    description: 'Python binary to be used: x64 for Mac or x86 for Windows'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
```

```suggestion
  steps:
    - run: echo "Can't run on Linux!"; exit 1
       if: ${{ runner.os == "Linux" }} 
```
Stop this action from running on Linux (it shouldn't)